mofacts/client/views/home/profile.js

```diff
@@ -488,6 +488,18 @@ async function selectTdf(currentTdfId, lessonName, currentStimuliSetId, ignoreOu
     };
     await updateExperimentState(newExperimentState, 'profile.selectTdf');
 
+    const curTdf = await meteorCallAsync('getTdfById', currentTdfId);
```

Comment:
There's actually already a function for this: https://github.com/memphis-iis/mofacts-ies/blob/postgresMigration/mofacts/client/views/experiment/card.js#L864